packages/sdk/contractkit/src/wrappers/SortedOracles.ts

```diff
@@ -233,13 +268,12 @@ export class SortedOraclesWrapper extends BaseWrapper<SortedOracles> {
 
   /**
    * Gets all elements from the doubly linked list.
-   * @param token The CeloToken representing the token for which the Celo
-   *   Gold exchange rate is being reported. Example: CeloContract.StableToken
+   * @param target The ReportTarget, either CeloToken or currency pair in question
    * @return An unpacked list of elements from largest to smallest.
    */
-  async getTimestamps(token: CeloToken): Promise<OracleTimestamp[]> {
-    const tokenAddress = await this.kit.registry.addressFor(token)
-    const response = await this.contract.methods.getTimestamps(tokenAddress).call()
+  async getTimestamps(target: ReportTarget): Promise<OracleTimestamp[]> {
+    const identifier = await this.toCurrencyPairIdentifier(target)
```

Comment:
I think you want to add curly braces and extract the `identifier` property:

```
const { identifier } = await this.toCurrencyPairIdentifier(target)
```

Reply:
The `toCurrencyPairIdentifier` function actually returns the identifier, because it's meant to work both with a `CeloToken` or a `PairWithidentifier` as argument. Thus for the `CeloToken` it looks up the address in the registry and for the `PairWithidentifier` it returns the identifier from the object.

Reply:
whoops. Totally misread that!